packages/lesswrong/components/ultraFeed/UltraFeedPostDialog.tsx

```diff
@@ -568,16 +575,21 @@ const UltraFeedPostDialog = ({
               </div>
               <div className={shouldShowToc ? classes.dialogInnerWrapper : undefined} ref={shouldShowToc ? scrollableContentRef : undefined}>
                 {shouldShowToc && (
-                  <div className={classes.tocColumnWrapper}>
-                    {hasTocData && tocData && (
-                      <FixedPositionToC
-                        tocSections={tocData.sections}
-                        title={displayPost.title}
-                        heading={<PostFixedPositionToCHeading post={displayPost as PostsListWithVotes}/>}
-                        scrollContainerRef={scrollableContentRef as React.RefObject<HTMLElement>}
-                      />
-                    )}
-                  </div>
+                  <>
+                    {/* placeholders for side comments, reacts, and notes */}
+                    <div />
```

Comment:
?

Reply:
used to match the grid layout of PostsPage.tsx and get good centering

Reply:
ah.  comment should indicate that it's for the grid, then (though I'm now a little 🤔 about the whole thing)